src/main/java/og_spipes/persistence/dao/ScriptDAO.java

```diff
@@ -113,7 +113,7 @@ public List<Module> getModules(Model m) {
             }else{
                 LOG.error("No type found for module: " + module.getUri());
             }
-            File scriptFile = uriFileMap.get(module.getUri());
+            File scriptFile = uriFileMap.get(normalizeUri(module.getUri()));
```

Comment:
Why we need this and why we call it normalizeUri ?

Reply:
hopefully fixed in https://github.com/kbss-cvut/s-pipes-editor/compare/c330ddd047362534fc1cf13cea633d7ed2b550b6..0e39e2f916faed13aaa8e0fcd11ef2589e707bc2